sem_cube.py

```diff
@@ -0,0 +1,140 @@
+from cmath import exp
+
+
+data_types = {
+    'int': 'int',
+    'float': 'float',
+    'char': 'char',
+    'bool': 'bool',
+    'arr1d': 'arr1d',
+}
+
+class sem_cube:
+    valid_ops = {
```

Comment:
```suggestion
    valid_operations = {
```
O puede ser:
```suggestion
    valid_expressions= {
```

Elegir uno para que tenga consistencia con los métodos.